pr_watcher_notifier/notification.py

```diff
@@ -26,6 +26,9 @@ def make_email(data):
         recipients=context['to'],
     )
     msg.body = body
+    current_app.logger.info('Sending email to {!r} with subject {!r}'.format(
+        context['to'], subject,
+    ))
```

Comment:
I don't like a log message that says "Sending" in a function that doesn't send.

Reply:
@nedbat, even before the recent change which added this line, [this line](https://github.com/open-craft/pr_watcher_notifier/blob/master/pr_watcher_notifier/notification.py#L16) added in https://github.com/open-craft/pr_watcher_notifier/commit/4e2afbf7ed030cfbe9aa94269ea3b10b1af3d253 already logs a similar message in the same function. At the time of that commit, the function was still named `send_email()`, so it made sense.

Will fix that as well :)